src/Interfaces.jl

```diff
@@ -202,19 +245,29 @@ macro interface(IT, alias_or_block, maybe_block=nothing)
     return esc(quote
         Interfaces.isinterfacetype(::Type{$IT}) = true
         Interfaces.interface(::Type{$IT}) = $iface
-        function Interfaces.implements(::Type{T}, ::Type{$IT}, mods::Vector{Module}=[parentmodule(T)]) where {T}
+        function Interfaces.implements(::Type{T}, ::Type{$IT}, mods::Vector{Module}=[parentmodule(T)]; debug::Bool=false) where {T}
+            debuglvl = debug ? :error : debug
             $block
         end
     end)
 end
 
-macro implements(T, IT)
+macro implements(T, IT, debug=false)
+    x = debug isa Bool ? debug :
+        (debug isa Expr && debug.head == :(=)) ? debug.args[2] :
+        throw(ArgumentError("unsupported argument in @implements: `$debug`"))
```

Comment:
this will handle `@implements T IT true` and `@implements T IT debug=true`, which i think we probably want... but this is the simplest way i could find to support both (and probably not the most robust!)

Reply:
LGTM